README.md

```diff
@@ -64,8 +63,10 @@ In this wave we will extend our `QuoteView` class to support clicking on the Buy
     * Decrease the stocks' price by a fixed amount (say, $1.00).
```

Comment:
I would just give them a fixed amount to add by, easier that way.

Adding this as a business logic method would also give us something to test!